EDU-18163: FastStore flag useIsGiftFromOrderForm (#2623)
vtexdocs/dev-portal-content: docs/faststore/docs/developer-tools/custom-cypress.mdx

> ⚠️ Although exclusive execution of custom tests is possible for projects using Cypress version 12.x, we do not recommend this approach. If possible, keep the default Cypress tests.

> Some projects may prefer exclusive execution of their custom Cypress tests. Although it is not our recommendation, for projects using Cypress version 12.x or higher, there is the possibility of running the **Integration Tests** check only for your custom tests.

Reviewer: ```suggestion
Some projects may prefer exclusive execution of their custom Cypress tests. Although it's not our recommendation, for projects using Cypress 12.x or higher, you can run the **Integration Tests** only for your custom tests.
```
